test/README.md

```diff
@@ -534,8 +534,12 @@ The environments files are located in the `envs` directory.
 - `regional-dr.yaml` - for testing regional DR using a hub cluster and 2
   managed clusters with Ceph storage.
 
-- `regional-dr-external.yaml` - A starting point for creating
+- `regional-dr-external.yaml.example` - A starting point for creating
    environment for testing regional DR using with external storage.
+   You need to implement "my-external-storage" component by
+   deploying the external storage drivers in
+   my-external-storage/start hook before using this yaml file.
+   You should rename it to regional-dr-external.yaml when you use it.
```

Comment:
I don't think we need the additional text, it is already explained in the file.
If someone use this example they will probably create another file and it will 
not be part of ramen source, so "should renamse it to `regional-dr-external.yaml`
is not a very useful advice.

Updating the name in the readme is good enough and it best squashed into the previous
commit since it is part of the same change.